google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/stub/EnhancedBigtableStubSettings.java

```diff
@@ -245,13 +236,7 @@ public boolean isRefreshingChannel() {
         .setKeepAliveTimeout(
             Duration.ofSeconds(10)) // wait this long before considering the connection dead
         .setKeepAliveWithoutCalls(true) // sends ping without active streams
-        // TODO(weiranf): Set this to true by default once DirectPath goes to public beta
-        .setAttemptDirectPath(isDirectPathEnabled());
-  }
-
-  // TODO(weiranf): Remove this once DirectPath goes to public beta
-  private static boolean isDirectPathEnabled() {
-    return Boolean.getBoolean("bigtable.attempt-directpath");
+        .setAttemptDirectPath(true);
```

Comment:
Can you add a  comment on why we attemt DirectPath? to give context to future readers.

Reply:
Added comment. PTAL at the wording, thanks!